src/cyclonedds/tests/test_ddsls.py

```diff
@@ -543,5 +543,5 @@ async def test_inner():
 
 
 def test_file_open_error():
-    data = run_ddsls(["--json", "-a", "--filename", "C:/test"])
-    assert "Exception: Could not open file C:/test" in data["stderr"]
+    data = run_ddsls(["--json", "-a", "--filename", "C:/this/path/denfinitely/doesnot/exist/ever"])
+    assert "Exception: Could not open file C:/this/path/denfinitely/doesnot/exist/ever" in data["stderr"]
```

Comment:
Can you replace this with `tmp_path` and use a non-existent directory under that path? It's highly unlikely `C:/this/path/denfinitely/doesnot/exist/ever` exists, but it's hardly a guarantuee :slightly_smiling_face:. With a non-existent directory beneath a tmp_path, we're certain.